GG/src/DeferredLayout.cpp

```diff
@@ -0,0 +1,82 @@
+/* GG is a GUI for SDL and OpenGL.
+   Copyright (C) 2003-2008 T. Zachary Laine
```

Comment:
I think if you're adding a new file, you shouldn't but Zach's copyright notice on it...